tests/Unit/UserTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
Rodei seus testes aqui e somente um passou. Comecei a fazer algumas alterações mas percebi que seria importante fazermos uma chamada em algum horário alternativo para conversarmos.

Apenas para que fique ciente nesse primeiro momento:

Ao invés de fazer um post para a rota você poderia usar uma factory. 

$response = $this->get('/api/users'); o correto aqui seria usar acting as $response = $this->actingAs($user)->get('/api/users'); assim ele iria agir como um usuário logado.

 Todas as variáveis estão como $response mas quando criamos um usuário deveria ser $user. 

Apesar de login ser um post, retornamos 200 ao invés de 201.

Creio que os nomes poderiam ser mais descritivos, exemplo, ao invés de test_check_bad_request_login_api_response poderia ser test_user_cannot_login_with_invalid_credentials;

Reply:
Os testes unitários foram finalizados e corrigidos.